### PR TITLE
Update timer randomized delay

### PR DESCRIPTION
**Pull request by ngetahun**
## Description

* Related Issue / Ticket / Trello card: <link reference>

## Change Type

*Please select the correct option.*

- [ ] **Bug Fix** (a non-breaking change which fixes an issue)
- [ ] **New Feature** (a non-breaking change which adds new functionality)
- [x] **Documentation Update** (a change which only updates documentation)

## Checklist

*Please check off each item if the requirement is met.*

- [ ] I have reviewed my own code and believe that it's ready for an external review.
- [ ] I have provided comments for any hard-to-understand code.
- [ ] I have documented the `MANUAL.md` file with any changes to the user experience.
- [ ] If my changes are non-trivial, I have added a changelog entry to notify users at `package/obs/rmt-server.changes`.

## Review

Please check out our [review guidelines](https://github.com/SUSE/scc-docs/blob/master/team/workflow/code_review.md) 
and get in touch with the author to get a shared understanding of the change.

### CODE REVIEW

**Review by felixsch — APPROVED** [review]
lgtm :+1:

File: package/obs/rmt-server.changes [hunk 1]
```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Wed Oct 30 09:01:32 UTC 2024 - Natnael Getahun <natnael.getahun@suse.com>
+
+- Version 2.20 (unreleased)
```
Comment (digitaltom) [inline package/obs/rmt-server.changes:4]: I think it also includes Adnilson's schema changes.

**Comment by ngetahun** [comment]
[[REPCT] RMT packaging: make sync/mirror timer changes survive upgrade](https://trello.com/c/VnmjxEIF/3522-repct-rmt-packaging-make-sync-mirror-timer-changes-survive-upgrade)

File: package/obs/rmt-server.changes (hunk 1 above)
Comment (jesusbv) [reply package/obs/rmt-server.changes:4]: yes, and, at least, these PRs (for the Pubcloud)
#1225
#1226

File: package/obs/rmt-server.changes [hunk 2]
```diff
@@ -1,3 +1,9 @@
...
+  * RMT packaging: make sync/mirror timer changes survive upgrade
```
Comment (digitaltom) [inline package/obs/rmt-server.changes:5]:
```suggestion
  * RMT packaging: don't overwrite custom sync/mirror timer config on package update
  * Extend column size for repository and file paths (bsc#1229152)
```

**Review by felixsch — APPROVED** [review]
lgtm :+1:

**Pull request merged**